package.json

```diff
@@ -108,6 +108,9 @@
     "typescript": "^3.9.3",
     "yarn-deduplicate": "^3.1.0"
   },
+  "resolutions": {
+    "react-resize-detector/**/lodash": "^4.17.21"
```

Comment:
I like this 
we maybe want to do the same for the other libs depending on lodash? 
Did you check if they had a new version not using lodash .20

Reply:
You're right. I updated the PR.